Card.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Text;
+
+namespace Card_Game
+{
+    public class Card : IComparable<Card>
+    {
+       
+        private Dictionary<CardsAttributes, int> attributes;
+        public int ID { get; private set; }
+        public string Name { get; private set; }
+        public string Description { get; private set; }
+
+        public int this[CardsAttributes key]
+        {
+            get
+            {
+                return attributes[key];
+            }
+        }
+
+        public Card(int ID, string name, int[] attributes, string description)
+        {
+            this.ID = ID;
+            Name = name;
+            Description = description;
+            this.attributes = new Dictionary<CardsAttributes, int>
+            {
+                {CardsAttributes.Fluffiness, attributes[0]},
```

Comment:
How about we use a loop over enum here to make it more flexible in case there are more attributes in the csv file and enum?
e.g. foreach(Foos foo in Enum.GetValues(typeof(Foos)))

Reply:
It has been changed